src/06-loops.js

```diff
@@ -151,7 +211,12 @@ function lastIndexFound(array, target) {}
  *  //> [ 7 * 0, 10 * 1, 11 * 2 ]
  *  //> [ 0, 10, 22 ]
  */
-function timesIndex(array) {}
+function timesIndex(array) {
+  array.forEach((e,i) => {
```

Comment:
nice `forEach` - just make sure you know that the  what each argument is actually doing under the hood